CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.38.2 (2020-10-21)
+
+* Sets the new `forbiddenFields` option to an empty array.
```

Comment:
for the Manage Workflow view, a "virtual" piece type which has the unusual property of viewing content of all types.